src/views/Home/utils/types.ts

```diff
@@ -4,3 +4,14 @@ export interface ContributorsInformation {
   contributors: number;
   description: string;
 }
+
+export interface MilestoneInformation {
+  title: string;
+}
+
+export interface RoadmapInformation {
+  id: string;
+  title: string;
+  description: string;
+  milestones: MilestoneInformation[];
+}
```

Comment:
You should reuse the `Milestone` or `Roadmap` types instead as this is the type coming from the API. Take a look into this file `src\core\models\interfaces\roadmaps.ts`

Reply:
Done